src/sparse/merkle_tree.rs

```diff
@@ -527,4 +548,80 @@ mod test {
         let expected_root = "e912e97abc67707b2e6027338292943b53d01a7fbd7b244674128c7e468dd696";
         assert_eq!(hex::encode(root), expected_root);
     }
+
+    #[test]
+    fn test_load_returns_a_valid_tree() {
+        // Instantiate a new key-value storage backing and populate it using a sparse Merkle tree.
```

Comment:
Can you break comments at 80 characters? I use side-by-side diffs and breaks at where you're doing it result in really short second lines that are hard to read.

Reply:
Since there are a number of comments in this file, I will create an issue to change the comment line length. It looks like this can be accomplished by specifying it in `rustfmt.toml` and running `cargo fmt`. 

Created issue https://github.com/FuelLabs/fuel-merkle/issues/85.